src/chainparams.cpp

```diff
@@ -96,6 +96,9 @@ class CMainParams : public CChainParams {
         consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000003404ba0801921119f903495e");
         consensus.defaultAssumeValid = uint256S("0x00000000000000000009c97098b5295f7e5f183ac811fb5d1534040adb93cabd"); // 751565
 
+        // Defer full RBF activation
+        m_defer_full_rbf =  1682899200; // 1 May 2023 00:00:00 +0000
```

Comment:
```suggestion
        m_defer_full_rbf =  1698796800; // 1 Nov 2023 00:00:00 +0000
```

Reply:
> If you have more ideas of another way forward to deploy full-rbf, I would invite you to open a code proposal. Or if you have conceptual reserves on the idea of full-rbf deployed by Bitcoin Core, I would invite you to pursue the conversation on the mailing list.

@ariard This is the only change I would like in this pull request and prefer this over other 2 for fullrbf(default) in bitcoin core.

Already sent an email to bitcoin-dev, not updated yet.

Reply:
I think the date should be informed by either (a) how long it will take anyone currently relying on opt-in signalling for accepting unconfirmed txs to change their software or business; and/or (b) the development timeline of any new protocols that rely on always being able to do rbf. Would be happy to change it if people actively involved in either of those things want to speak up.